Formula/gopass.rb

```diff
@@ -0,0 +1,43 @@
+class Gopass < Formula
+  desc "The slightly more awesome Standard Unix Password Manager for Teams"
+  homepage "https://www.justwatch.com/gopass"
+  url "https://www.justwatch.com/gopass/releases/1.2.0/gopass-1.2.0-darwin-amd64.tar.gz"
```

Comment:
This will need to be built from source, as is standard in Homebrew core. Looks pleasantly simple for this particular project.